pkg/assets/assets.go

```diff
@@ -0,0 +1,67 @@
+package assets
+
+import (
+	"fmt"
+	"io/ioutil"
+	"os"
+	"path/filepath"
+
+	"github.com/coreos/bootkube/pkg/assets/internal"
+)
+
+const (
+	AssetPathCAKey           = "ssl/ca.key"
```

Comment:
Can we generate the same asset names between bootkube, bootcfg, and coreos-kubernetes? botocfg follows the same conventions as coreos-kubernetes, which differs from these. https://github.com/coreos/coreos-baremetal/tree/master/examples#assets